jcl/source/windows/JclTD32.pas

```diff
@@ -1391,6 +1397,44 @@ procedure TJclTD32InfoParser.AnalyseUnknownSubSection(const pSubsection: Pointer
   // do nothing
 end;
 
+function TJclTD32InfoParser.FormatProcName(const ProcName: string): string;
+var
+  pchSecondAt, P: PChar;
```

Comment:
Do not use hungarian notation for variables
Give meaningful names to variables (P is not )